src/sdk/namespace_fs.js

```diff
@@ -2320,6 +2369,8 @@ class NamespaceFS {
             delete_marker,
             storage_class,
             restore_status: GlacierBackend.get_restore_status(stat.xattr, new Date(), this._get_file_path({key})),
+            multipart_size,
```

Comment:
why do we return multipart_size? where it's being used?

Reply:
probably my idea was to use it in s3_utils.set_response_object_md() but eventually I didn't.